corehq/apps/api/resources/v0_5.py

```diff
@@ -57,6 +57,18 @@ def user_es_call(domain, q, fields, size, start_at):
     return query.run().hits
 
 
+def _set_role_for_bundle(kwargs, bundle):
+    # check for roles associated with the domain
+    domain_roles = UserRole.by_domain_and_name(kwargs['domain'], bundle.data.get('role'))
+    if domain_roles:
+        qualified_role_id = domain_roles[0].get_qualified_id()
```

Comment:
this is for duplicate names? might be worth a comment mentioning that the user just gets the first one.